plugins/basic/nexus-repository-httpbridge/src/main/java/org/sonatype/nexus/repository/httpbridge/internal/describe/DescribeType.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.httpbridge.internal.describe;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Describe type.
+ *
+ * @since 3.0
+ */
+public enum DescribeType
+{
+  HTML,
+  JSON;
+
+  /**
+   * Parse type from flags.  Either explicit HTML or JSON, or anything else will default to HTML.
```

Comment:
inaccurate. null will throw, and value is trimmed and made case insensitive by this method before explicit match.